31-temperature-and-smoke-detector/README.md

```diff
@@ -19,72 +22,63 @@ Temperature monitoring and smoke detection across a number of rooms at a facilit
 
-1. Splice a male to female jumper in half and solder the exposed wires to the `GND` pad on the 5v converter. Connect the female end to `GND` on the sensor and the male end to the `GND` pin on Sparrow.
-2. Splice another jumper, and solder the exposed wire on male half to `Vi` on the converter. Connect the male connector to `VBAT` on Sparrow. Solder the exposed wire on the female half to `Vo` and connect the female end to the `A0` pin on the MQ2 sensor.
-3. Using an intact male-to-female jumper, connect `A0` on the sensor to the `A1` pin on Sparrow.
-4. The 4th pin on the sensor is left unconnected.
+1. Follow the steps in the [Notecard Quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#connect-your-notecard-and-notecarrier) to connect your Notecard LoRa to your Notecarrier-F.  Your antenna will only have one cable, and the Notecard LoRa only has one connection.
 
-## Firmware
+2. Plug your Swan into the Feather headers on the Notecarrier F.
 
-This section shows how you build the Temperature and Smoke Detector firmware and flash it to the reference node.
+3. Attach the Swan to your computer with a Micro USB to USB-A cable, using the **Micro USB port on the Swan**.
 
-1. You will need to fetch the dependencies required by the firmware. After cloning this repository, run these commands: `git submodule update --init 31-temperature-and-smoke-detector/firmware/note-c` and `git submodule update --init 31-temperature-and-smoke-detector/firmware/sparrow-lora`. This will fetch the `note-c` and `sparrow-lora` submodules that this project depends on.
+![Notecard LoRa in Notecarrier F with antenna](images/notecarrier-notecard.jpg)
 
-2. Connect the STLINK-V3MINI to your development PC with a USB A to micro USB cable.
+### MQ2 Sensor Connection
 
-3. Connect the STLINK to your reference node with the 2x7 JTAG ribbon cable.
+This solution makes use of the an MQ2 sensor module, which detects smoke, as well as various gasses. Using the 3 male-to-female jumper wires, connect the MQ2 sensor to the Sparrow Reference Sensor board and the v5 converter as follows:
 
-4. There are a few ways to build and flash the firmware onto the reference node. These are covered in the [Sparrow Builder's Guide](https://dev.blues.io/sparrow/sparrow-builders-guide/). Follow the steps in that guide and then return to these instructions.
+1. Solder a set of headers onto the 5v converter.
+1. Connetor a jumper to `GND` pad on the 5v converter and the other end  to a `GND` pin on the Notecarrier-F.
+1. Connect a jumper to `GND` on the sensor and the other end  to a `GND` pin on the Notecarrier-F.
+2. Connect a jumper to `Vi` on the converter and the other end to  `F_3V3` on the Notecarrier-F. 
+2. Connect a jumper to `Vo` on the converter and the other end to the  `VCC` pin on the MQ2 sensor.
+3. Connect a jumper to `A0` on the sensor and the other end to the `F_A0` pin on the Notecarrier-F.
+4. The 4th pin on the sensor is left unconnected.
```

Comment:
You can start all of these with `1.`

Currently you have 1, 1, 2, 2, 3, 4 :)

Reply:
That's what happens when you decide to make the instructions make more sense and add steps!! I'll change them all :)